nix/default.nix

```diff
@@ -28,7 +28,7 @@ let
   docsPkgs = [
     pkgs.plantuml
     pkgs.texlive.combined.scheme-full
-    (pkgs.python3.withPackages
```

Comment:
this is because nixpkgs not too long ago bumped the default python version and now many python packages are broken